test/board_test.exs

```diff
@@ -26,4 +26,24 @@ defmodule BoardTest do
 
     assert board == expected_board
   end
+
+  test "determines noone has won" do
+    board = %{
+      1 => "X", 2 => "2", 3 => "X",
+      4 => "O", 5 => "5", 6 => "O",
+      7 => "X", 8 => "8", 9 => "O"
+    }
+
+    assert Board.has_player_won(board) === false
+  end
+
+  test "determines X is the winner" do
```

Comment:
Maybe explain in the test description that `has_player_won?` finds a winning diagonal. I would also expect to see tests around winning rows/columns. If you delete the production code checking for winning rows/columns do any tests fail?